packages/backend/src/lib/xpHelpers.ts

```diff
@@ -0,0 +1,25 @@
+export const getNumWeeksInSeason = (): number => {
+  const currentDate = new Date();
+
+  const currentYear = currentDate.getFullYear();
+
+  const lastYear = currentDate.getFullYear() - 1;
+
+  const seasons = [
+    new Date(Date.parse(`${lastYear}-12-21T00:00:00+00:00`)), // 21 Dec last year
+    new Date(Date.parse(`${currentYear}-03-20T00:00:00+00:00`)), // 20 Mar this year
+    new Date(Date.parse(`${currentYear}-06-21T00:00:00+00:00`)), // 21 June this year
+    new Date(Date.parse(`${currentYear}-09-22T00:00:00+00:00`)), // 22 Sept this year
+    new Date(Date.parse(`${currentYear}-12-21T00:00:00+00:00`)), // 21 Dec this year
+  ];
+
+  const currentSeason = seasons.reduce(
```

Comment:
This is an interesting use of `reduce`. I was incredulous that it worked initially, but I finally got it. 😆